pdfgeneration/pdfgeneration.py

```diff
@@ -174,9 +174,13 @@ def run(self, options):
             # replace the values
             txt = txt.replace("${PATIENT_TOKEN}", options.patientId)
             txt = txt.replace("${PREDICTION_CLASSIFICATION}", classification_data['prediction'])
-            txt = txt.replace("${COVID-19}", classification_data['COVID-19'])
-            txt = txt.replace("${NORMAL}", classification_data['Normal'])
-            txt = txt.replace("${PNEUMONIA}", classification_data['Pneumonia'])
+            
+            predictionAnalysis = ""
+            for columnName in classification_data:
+              if (columnName != 'prediction') and (columnName != 'Prediction') and (columnName != '**DISCLAIMER**'):
+                predictionAnalysis += "<h3>{title}: <span>{prediction}</span></h3>".format(title = columnName, prediction = classification_data.get(columnName, None))
```

Comment:
```suggestion
                predictionAnalysis += "<h3>{title}: <span>{prediction}</span></h3>".format(title=columnName, prediction=classification_data.get(columnName, None))
```

["Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter"](https://www.python.org/dev/peps/pep-0008/#other-recommendations)